build/buildRunTimeImages.sh

```diff
@@ -15,6 +15,105 @@ source $REPO_DIR/build/__functions.sh
 source $REPO_DIR/build/__sdkStorageConstants.sh
 source $REPO_DIR/build/__stagingRuntimeConstants.sh
 
+# Load platform versions for runtimes
+source $REPO_DIR/build/__dotNetCoreRunTimeVersions.sh
+source $REPO_DIR/build/__nodeVersions.sh
+source $REPO_DIR/build/__phpVersions.sh
+source $REPO_DIR/build/__pythonVersions.sh
+
+# Get the specific platform version used for each runtime image to assist with future patching
+# e.g., for dotnetcore:7.0-debian-buster, we would retrieve 7.0.10 from the __dotNetCoreRunTimeVersions.sh file
+function getRuntimeTagVersion()
+{
+    if [ -z $1 ]
+    then
+        echo "Runtime platform name was not provided"
+        return 1
+    fi
+
+    if [ -z $2 ]
+    then
+        echo "Runtime platform version was not provided"
+        return 1
+    fi
+
+    PLATFORM_NAME=$1
```

Comment:
In order to use these two args, do we need to modify the pipeline templates we have currently? Seems like the current `_buildTemplate.yml` doesn't have these two parameters yet.

Reply:
These arguments are scoped within the `getRuntimeTagVersion` function I added -- I call `getRuntimeTagVersion` later in this file and pass the `$platformName` and `$platformVersion` variables to it.